extensions/blocks/donations/donations.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Donations Block.
+ *
+ * @since 8.x
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Donations;
+
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'donations';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block(
+		BLOCK_NAME,
+		array( 'render_callback' => __NAMESPACE__ . '\load_assets' )
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
```

Comment:
I think we usually hook into `jetpack_register_gutenberg_extensions` for registering the Jetpack blocks. Will look into it.

Reply:
`init` works here 👍

Reply:
I used the `wp jetpack scaffold` command here to generate the init block boilerplate, but let us know if there's a preferred style.